### ISSUE TITLE
Do at your own risk.

### ISSUE THREAD

**Issue by Rab89** [finndev/PokeBuddy#996]
As you are aware that a ban wave seems pretty imminent from niantic so continue at own risk, alot of other bots seems to be having legal threats from niantic and forcing them to pull the plugs on their bots.
i also think they have extended bans ie soft bans and ip bans.
Quickspin no longer works i can confirm.
workaround is ensure you walk speed is between 5-10km/h alot slower but steadier.

-**IF IT STILL DOESNT WORK FOR YOU THATS DUE TO STILL BEING BANNED**.

**Comment by JavierBarrrera** [comment]
Agree.

**Comment by karimavine** [comment]
Is "failed to get data from the server" is a perm ban ?  my pokemons are still in a gym :"D

**Comment by Rab89** [comment]
@karimavine the pokemon go servers have been unsteady all day. id keep trying the smartphone app,

**Comment by karimavine** [comment]
@Rab89 it`s working with another account i just created

**Comment by AlexCtasy** [comment]
@karimavine Maybe is a perma ban, but strange that not show banned or something like that, try log ur acocunt at PTC website.

**Comment by MJH94** [comment]
Quickspin does work on my secondary account i just made td. Have been using necrobot for a few hours no problem with the quickspin working when it needs to.  Already lvl 23 no problems

**Comment by Rab89** [comment]
like i said servers arent steady  @karimavine , as far as im aware perma bans havn't been issued, were currently all having issues with the game.

**Comment by MJH94** [comment]
100% fact quickspin does work tho

**Comment by karimavine** [comment]
@AlexCtasy im registered with google account  is there anyway i can log in in ptc ? to check if it`s perm

**Comment by Rab89** [comment]
@MJH94 Necrobot is one of the bots that is pulling the plug

**Comment by AlexCtasy** [comment]
@MJH94 how u lv up like that from 0, What bot is that? pokebuddy? or another one}?

**Comment by MJH94** [comment]
ok Well i have been running necrobot all day no problems

**Comment by MJH94** [comment]
just cant download it anymore. if you have it, it works

**Comment by MJH94** [comment]
Went from lvl 0 to 23 in 2 hours with necro today

**Comment by AlexCtasy** [comment]
@mjh94 can u give me the link to download that necrobot?

**Comment by MJH94** [comment]
They removed it earlier today

**Comment by MJH94** [comment]
I already had it on my com

**Comment by Rab89** [comment]
upload it mediafire for him lol sharings caring :stuck_out_tongue:

**Comment by AlexCtasy** [comment]
Yes please :D

**Comment by MJH94** [comment]
ok give me about 5 min guys. will have most recent version of necrobot for you

**Comment by AlexCtasy** [comment]
Oka

**Comment by MJH94** [comment]
#998

**Comment by imbot87** [comment]
thanks mate!

**Comment by imbot87** [comment]
works perfectly :D

**Comment by jacearchitect69** [comment]
i cannot read the link of necrobot mediafire can somebody pm it to me too?

**Comment by randomerrorgenerator** [comment]
So I've been using this bot recently , and see where everyone is coming from, they've defently put a soft ban on people going over walking speeds 6km is fine, unless you can click the pop up for going faster. When I get soft banned I use nox, and run around on that, which oddly I can use for hours on end, but obviously extremely manual. I think we need some kind of human error and no 100% catches and some other stuff in play. It makes it pretty obvious, and probably some lag in between interactions like catching and moving and depositing.

**Comment by kondziu124** [comment]
Yep. the issue is that bots are too fast, the bot speed should not exceed a normal, walking one when left for a couple of hours running. Also, 100% catch chance, even when it uses a few pokeballs is a problem, bots often tend to catch a Poke and then in an instant deposit, move, use Pokestop or so. 
Using nox manually won't result in a ban (just bear in mind not to teleport far away and rather move for not more than a 100meters, than take a break, than move again and so on). I've been using nox on my main acc since it came out that nox can be used to play PoGo, change locations etc., without a ban, block, softan and such. But then, started botting and now I'm left with a ban(?) or maybe not, no e-mails from Niantic, just this data error and Pokebuddy doesn't make a thing, doesn't show anything, doesn't move when logged onto my account. It has to be something regarding accounts DIRECTLY not IPs, I've tried alot of different VPNs, but nothing's changed, still had that data error after launching the game via Nox and still, bot did nothing and showed nothing (even my lvl was showed as 0) so this may be somehow connected to that data error. Either this is just some another Niantic servers shit or they've started a massive banwave for botting people, blocking that data or erasing it completely.
